src/main/resources/public/jsx/contact.jsx.js

```diff
@@ -175,6 +186,13 @@ var ContactForm = React.createClass({
                             <ContactCheckboxField onChange={this.handleChange} renderLabel={this.renderLabel}
                                                   name="copyToSender" labelName="copy-to-sender" value={this.state.fields.copyToSender}
                                                   errors={this.state.errors} />
+                            <ContactCaptcha     sitekey="6LexYCkUAAAAADTNXhV0ZkY7eYJrHnOxvOBTERCw"
```

Comment:
I can create a config file `config.json` in [public folder](https://github.com/ozwillo/ozwillo-portal/tree/master/src/main/resources/public)
```
{
 "example": ""
}
```
And add external config files to webpack.base.babel.js file like this:
```
  externals: {
    config: JSON.stringify(require(PATH.app + '/config.json')), 
  }
```
Then we can use it like this:
```
ìmport config from 'config';

console.log(config.example);
```
what do you think ?